webauthn-authenticator-rs/examples/authenticate/main.rs

```diff
@@ -118,8 +145,17 @@ fn main() {
     let cred = wan.register_credential(&r, &reg_state, None).unwrap();
 
     trace!(?cred);
+    drop(u);
+    let mut buf = String::new();
+    println!("WARNING: Some NFC keys need to be power-cycled before you can authenticate.");
```

Comment:
Doesn't have to be in this PR, maybe we should check the AAGUID of the device get_info and only issue this warning if it's a known problem device?

Reply:
The specific case this is handling is someone leaving an NFC key placed on the reader.

The Windows WebAuthn API enforces this (on a software level) for all requests -- once the CTAP backend here gets some more polish I'd like to have it work the same way.

The Feitian keys are the only NFC ones I've seen enforce this in firmware.  I haven't actually figured out whether this is another bug in Feitian's firmware, because right now for NFC keys I can test, it's Feitian vs. Yubikey, and I'll need to read the CTAP spec again to figure out if they got ideas from there.  Seeing what a third vendor does would help.

The Feitian key doesn't seem to care if you repeatedly authenticate without moving the key. It only needs a power-cycle after registration... which makes me think that it may not be intentional.

One of the things I'd like to tie into some MDS work is actually document some of these sorts of issues; because from what I can tell the current FIDO stuff relies a lot on libraries shipping work-arounds.